Ginger/Ginger/ApplicationModelsLib/APIModels/APIModelWizard/AddAPIModelWizard.cs

```diff
@@ -42,7 +42,9 @@ public enum eAPIType
             [EnumValueDescription("JSON Templates")]
             JsonTemplate,
             [EnumValueDescription("Swagger(Open API) Document")]
-            Swagger
+            Swagger,
+            [EnumValueDescription("YAML")]
```

Comment:
Please remove YAML option from here as YAML file can be uploaded via swagger document